imagebuildah/build.go

```diff
@@ -227,6 +244,46 @@ func (b *Executor) Preserve(path string) error {
 	return nil
 }
 
+// getSymbolicLink finds any symbolic links in the path
+// If the path is "/my/path/with/symlinks" and "/my/path" is the symlink
+// getSymbolicLink starts with "/my" then "/my/path" and so forth to find a symlink
+// if one exists. Returns an empty string if a symlink doesn't exist
+func getSymbolicLink(mountPoint, path string) (string, string, error) {
```

Comment:
Did you look at https://golang.org/pkg/path/filepath/#EvalSymlinks, does everything you need except you are still dangerous.  
We have to do these calls inside of a chroot.

Look at libpod/pkg/chrootuser for examples of how to do this.